tracer.go

```diff
@@ -205,6 +213,10 @@ func (t *Tracer) startSpanWithOptions(
 		if hasParent && parent.isDebugIDContainerOnly() {
 			ctx.flags |= (flagSampled | flagDebug)
 			samplerTags = []Tag{{key: JaegerDebugHeader, value: parent.debugID}}
+			textMapPropagator, ok := t.injectors[opentracing.TextMap].(*textMapPropagator)
+			if ok {
+				samplerTags = []Tag{{key: textMapPropagator.headerKeys.JaegerDebugHeader, value: parent.debugID}}
```

Comment:
it is not critical for the tag to match the name of the header, this addition is not necessary.